js/ai/src/generate.ts

```diff
@@ -346,6 +346,8 @@ async function resolveFullToolName(
     return `/tool/${name}`;
   } else if (await registry.lookupAction(`/prompt/${name}`)) {
     return `/prompt/${name}`;
+  } else if (await registry.lookupAction(`/prompt/dotprompt/${name}`)) {
+    return `/prompt/dotprompt/${name}`;
```

Comment:
Separate thing: I really want prompts to be able to have slashes in their name so that you can have a `prompts` directory with subdirectories. But the registry doesn't allow that right now even though I think it could without too much effort.

Reply:
Yes, I believe it's a trivial validation fix. we can do it.

Reply:
To clarify: did you do it already?

Reply:
No, I'll do it as a follow up. Probably along with async action registration.